### PR TITLE
IGVF-872-calculate-sources-in-multiplex

### PR DESCRIPTION
**Pull request by iyoungworth**

### CODE REVIEW

File: src/igvfd/upgrade/sample.py [hunk 1]
```diff
@@ -234,3 +234,14 @@ def sample_12_13(value, system):
     # default value for 'virtual' property.
     # The default value will be automatically populated.
     return
+
+
+@upgrade_step('multiplexed_sample', '1', '2')
+def sample_13_14(value, system):
```
Comment (gabdank) [inline src/igvfd/upgrade/sample.py:240]: i think we have to capture the popped values in "notes"

**Review by gabdank — CHANGES_REQUESTED** [review]
record the deleted values in notes

File: src/igvfd/upgrade/sample.py (hunk 1 above)
Comment (iyoungworth) [reply src/igvfd/upgrade/sample.py]: Got it, I updated the upgrade

File: src/igvfd/tests/test_upgrade_multiplexed_sample.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import pytest
+
+
+def test_multiplexed_sample_upgrade_1_2(upgrader, multiplexed_sample_v1):
+    value = upgrader.upgrade('multiplexed_sample', multiplexed_sample_v1, current_version='1', target_version='2')
```
Comment (gabdank) [inline src/igvfd/tests/test_upgrade_multiplexed_sample.py:5]: please compare the notes

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/tests/test_upgrade_multiplexed_sample.py (hunk 2 above)
Comment (iyoungworth) [reply src/igvfd/tests/test_upgrade_multiplexed_sample.py:5]: Done, added this test

**Review by gabdank — DISMISSED** [review]

**Pull request merged**